Update ACFloatingTextfield.swift

dav3harry opened the pull request:
Change framerror size height to 15, since current label text is cut

dav3harry committed:
Update ACFloatingTextfield.swift

ErAbhishekChandani commented:
Thanks! i will look forward to this ASAP.

On Wed, 5 Apr 2017 at 5:02 PM, dav3harry <notifications@github.com> wrote:

> Change framerror size height to 15, since current label text is cut
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/ErAbhishekChandani/ACFloatingTextfield/pull/16
> Commit Summary
>
>    - Update ACFloatingTextfield.swift
>
> File Changes
>
>    - *M* Source-Swift/ACFloatingTextfield.swift
>    <https://github.com/ErAbhishekChandani/ACFloatingTextfield/pull/16/files#diff-0>
>    (2)
>
> Patch Links:
>
>    -
>    https://github.com/ErAbhishekChandani/ACFloatingTextfield/pull/16.patch
>    -
>    https://github.com/ErAbhishekChandani/ACFloatingTextfield/pull/16.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ErAbhishekChandani/ACFloatingTextfield/pull/16>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ANMZ106rWR9Np-B068k0i1CdjEryxdSSks5rs3vegaJpZM4M0HOa>
> .
>
-- 
*Thanks & Regards*
*Er. Abhishek Chandani*

dav3harry committed:
Update ACFloatingTextfield.swift

I need to change the line color after error to go back to the original color after DidBegin method start.

ErAbhishekChandani commented on Source-Swift/ACFloatingTextfield.swift:
```diff
@@ -253,6 +253,7 @@ fileprivate extension ACFloatingTextfield {
         UIView.animate(withDuration: 0.2) {
             self.labelErrorPlaceholder?.alpha = 0;
             self.labelErrorPlaceholder?.frame = labelErrorFrame!;
+            self.bottomLineView?.backgroundColor = self.placeHolderColor;
```
I think this is incorrect , why changing the line background color with placeholder color , it has the seperate property for this, i can merge this request but this needs to be removed (line no 256) Thanks

dav3harry committed:
Update ACFloatingTextfield.swift

dav3harry committed:
Update ACFloatingTextfield.swift

dav3harry commented:
Remove the placeholder label color for error state
